Documentation/UseCases/Index.rst

```diff
@@ -272,3 +272,49 @@ like `https://example.com/downloads/detail/some-download/hz6dFgz9/`, where
 
 .. _custom dimension: https://matomo.org/docs/custom-dimensions/
 .. _setting a custom URL: https://matomo.org/faq/how-to/how-do-i-set-a-custom-url-using-the-matomo-javascript-tracker/
+
+Extending the script tag
+========================
+
+Using tracking tools like Matomo within the European Union need special treatments in order
+to let the customer consent and agree with the tracking. Even Matomo respects the "Do not track" setting from the browser not everybody
```

Comment:
Even Matomo respects the "Do not track" setting from the browser not everybody may be aware of it.

Had to read it twice to understand the sentence. Perhaps:

    Although Matomo respects the browser's "Do not track" setting, not everyone is aware of it.

Deepl is my favourite :-) First I write English, translate it to German and then back to English and look at the difference ;-)

Reply:
Description changed.